Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ruby:2.4.1
+
+LABEL maintainer Travis CI GmbH <support+travis-app-docker-images@travis-ci.com>
+
+RUN apt-get update && apt-get upgrade -y --no-install-recommends && apt-get install -y postgresql
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY Gemfile      /usr/src/app
+COPY Gemfile.lock /usr/src/app
+
+RUN bundle install --deployment
+
+COPY . /usr/src/app
+
+RUN apt-cache search postgres
+# Install sqitch so migrations work
+RUN env BINDIR=/usr/local/bin /usr/src/app/script/install-sqitch
```

Comment:
The database must be created separately, like it's being done in the `.travis.yml` (and activerecord migrations, for that matter) 👍.  As you've found, the sqitch config file only specifies a test database.  Sqitch will accept a database URL(-ish value) as a positional argument, and since we can get the database URL directly from Heroku, running sqitch migrations on databases like staging and production is usually done like this:

``` bash
./script/sqitch-heroku LOGS_DATABASE_URL travis-logs-staging deploy
```

Performing the same type of migration on a local database that's named `some_other_database` would look like:

``` bash
sqitch deploy db:pg:some_other_database
```